ios/brave-ios/Sources/Brave/Frontend/Settings/Debug/BraveCoreDebugSwitchesView.swift

```diff
@@ -173,6 +173,68 @@ private struct BasicPickerInputView: View {
   }
 }
 
+private struct CustomSwitchInputView: View {
+  @ObservedObject private var customSwitches = Preferences.BraveCore.customSwitches
+  @ObservedObject private var switchValues = Preferences.BraveCore.switchValues
+  @Environment(\.presentationMode) @Binding private var presentationMode
+
+  @State var key: String = ""
```

Comment:
Probably shouldn't initialize `State` like this, best to pass in a regular `var` and `onAppear` set the State with that. If you're fine with the `key` passed in being ignored on subsequent body evaluations you can also make a explicit init method that sets the state like `_key = State(initialValue: key)` instead